trieste/acquisition/function/function.py

```diff
@@ -798,3 +798,65 @@ def __call__(self, x: TensorType) -> TensorType:
         mean, variance = self._model.predict(x)  # [..., B, 1]
         mean, variance = tf.squeeze(mean, -1), tf.squeeze(variance, -1)
         return -mean + tf.sqrt(variance) * self._betas  # [..., B]
+
+
+class MakePositive(SingleModelAcquisitionBuilder[ProbabilisticModelType]):
```

Comment:
this class should be imported in `__init__` files in this dir and the parent one, right?